dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -20,11 +21,27 @@
        Otherwise, play it safe by using a temporary table.
```

Comment:
@McKnight-42 could you update the docs in this section?

Maybe add a line here like:
```
       Low-level specifics:
       If an invalid option is specified, then raise an error with an applicable explanation
```

And another update like:
```
       When the unique_key is none, the delete+insert strategy
       can use a view because it will run a single INSERT statement.
```